src/Umbraco.Core/Services/ContentTypeServiceBaseOfTRepositoryTItemTService.cs

```diff
@@ -329,6 +329,22 @@ public IEnumerable<TItem> GetAll(IEnumerable<Guid>? ids)
         }
     }
 
+    public Task<PagedModel<TItem>> GetAllAsync(IEnumerable<Guid> keys, int skip, int take)
```

Comment:
If we take multiple keys as input, we also need to return those keys, instead of paging

Reply:
So when the logic from the controller is moved to a service it makes sense to have paging again. But the current setup does not really make sense